src/reducers/toggleToolbarAndBreadCrumbs.ts

```diff
@@ -0,0 +1,10 @@
+import { State } from '../util/initialState'
+
+/** Toggles the Toolbar Visibility. */
+const toggleToolbarAndBreadCrumbs = (state: State, { value }: { value?: boolean }) => ({
+  ...state,
+  showToolbar: value === null ? !state.showToolbar : value,
```

Comment:
This is a bit more terse: `value ?? !state.showToolbar`

Reply:
Nullish coalescing, cool 😃

Reply:
I know, I just learned about it!

Reply:
You can use this technique for `showToolbar` as well.